contributing.md

```diff
@@ -5,23 +5,35 @@ This document explains how, technically, to contribute to this project. A code o
 ## Contribution Workflow
 
 * Check the issues and draft pull requests to see if anyone is working on something similar.
-![](assets/issues-prs.png)
 * Make a branch for your contribution.
-![](assets/pycharm-branch.png)
-* Implement your feature, bug fix, documentation, etc.
-  In pycharm:
-  * Add any new files to `git` (i.e. stage them):
-  ![](assets/stage.png)
-  * Commit the changes (with a sensible message).
-  ![](assets/commit.png)
-  > Add / commit multiple times to break your contribution into easy to follow chunks. 
-  * Push your changes:
-  ![](assets/push.png)
-* Make a pull request against the `master` branch of this repository.
-![](assets/pull-request.png)
+* Implement your feature, bug fix, documentation, etc. using commits.
+* Push your changes.
+* Make a pull request against the `master` branch of this repository for bug fixes, or the `dev` branch for new features.
 
 You are advised to make a draft pull request as soon as you start work so nobody else ends up working on the same thing.
 
+## Software Architecture
+
+`topostats` is currently arranged as a Python module in the `topostats/` folder. Other scripts in this repository are not currently considered core elements of `topostats` apart from `Plotting.py`.
+
+Currently the `topostats` module consists of:
+
+* `default_config.ini` The default config file.
+* `dnatracing.py`
+* `pygwytracing.py` The "main" routine.
+* `tracingfuncs.py`
```

Comment:
do dnatracing.py and tracingfuncs.py also need brief descriptors? Something like dnatracing - applies tracing functions to each molecules, and for tracingfuncs - skeletonises and generates backbone traces from masks (I think that's what they do at least)

Reply:
Good idea. I think these will be subject to further revisions as well.